client/src/lib/Sources/FeedLogViewer.svelte

```diff
@@ -96,6 +97,32 @@
     }
   };
 
+  const downloadFeedLogs = async () => {
+    if (!feed) {
```

Comment:
Maybe nit-picking but you could summarize the first two if-clauses to make the code a little more compact.

Reply:
Thanks :)